ui/app/templates/components/json-editor.hbs

```diff
@@ -1,32 +1,34 @@
-{{#if showToolbar }}
```

Comment:
this diff looks strange, but I think when this was last worked on the developer had their tab settings at 4. I changed the file tab settings to 2 to match the code base and this was the resulting diff. If you're nervous checkout the file in the branch and double check.

Reply:
I appreciate all of your notes! 😍